src/store.js

```diff
@@ -280,13 +286,29 @@ const store = new Vuex.Store({
     /* Called when app first loaded. Reads config and sets state */
     async [INITIALIZE_CONFIG]({ commit }) {
       // Get the config file from the server and store it for use by the accumulator
-      commit(SET_REMOTE_CONFIG, yaml.load((await axios.get('conf.yml')).data));
+      try {
+        commit(SET_REMOTE_CONFIG, yaml.load((await axios.get('conf.yml')).data));
+      } catch {
+        ErrorHandler('Unable to fetch conf.yml');
+      }
       const deepCopy = (json) => JSON.parse(JSON.stringify(json));
       const config = deepCopy(new ConfigAccumulator().config());
       commit(SET_CONFIG, config);
+      commit(SET_CONFIG_FETCHED, true);
+    },
+    /* Returns a promise which resolves when conf.yml has been fetched */
+    [WAIT_FOR_CONFIG]({ state }) {
+      return new Promise((resolve) => {
+        if (state.configFetched) resolve();
+        this.watch(() => state.configFetched, (value) => {
+          if (value) resolve();
+        });
+      });
     },
   },
   modules: {},
 });
 
 export default store;
+
+store.dispatch(INITIALIZE_CONFIG);
```

Comment:
Should come before `export default store;`, but maybe this should be in the `App.js`?
Edit: oh yeah, because store is being used in router too, keep it here then.